src/Selector/PartialNamedIncludingHiddenSelector.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/*
+ * This file is part of the Mink package.
+ * (c) Konstantin Kudryashov <ever.zet@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Behat\Mink\Selector;
+
+/**
+ * Named selectors engine. Uses registered XPath selectors to create new expressions.
+ *
+ * @author Konstantin Kudryashov <ever.zet@gmail.com>
+ */
+class PartialNamedIncludingHiddenSelector extends PartialNamedSelector
```

Comment:
Well, we need this in Drupal testing framework, see https://www.drupal.org/node/2767655. As you can see there, we init the session by adding also this new selector:

```
    $selectors_handler = new SelectorsHandler([
      'named_partial_including_hidden' => new PartialNamedIncludingHiddenSelector()
    ]);
    $session = new Session($driver, $selectors_handler);
```

Then we have an extension of WebAssert and override fieldExists() and fieldNotExists() to use also this new selector.

I was not thinking how this can be used in Behat scenarios. Probably that could be a follow up task?